host/4/bullseye/amd64/java/java21/java21-composite.template

```diff
@@ -0,0 +1,38 @@
+ARG JAVA_VERSION=21.0.0
+ARG JAVA_HOME=/usr/lib/jvm/msft-21-x64
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
+ARG HOST_VERSION
+ARG JAVA_VERSION
```

Comment:
Currently the host composite image is based on Bullseye so for now it likely makes more sense to replace this with an -appservice image. I can show Kirstyn how to do this. 
This is necessary to get Host on bookworm
@pragnagopa